src/main/java/seedu/address/logic/parser/ExportCommandParser.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.logic.commands.ExportCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new ExportCommand object.
+ */
+public class ExportCommandParser implements Parser<ExportCommand> {
+    private static final String ALL_KEYWORD = "all";
```

Comment:
It might be confusing/easily forgotten by the user that the default, argument-less behavior is to export only displayed `Persons`.

Perhaps allow an optional keyword to confirm this behavior, e.g. `export shown`? The default behavior with no arguments can still stay the same, i.e. goes ahead and exports only those displayed.
(but perhaps swapping default to exporting all would be more appropriate anw)